types/hellosign-sdk/index.d.ts

```diff
@@ -686,6 +653,31 @@ declare namespace HelloSign {
         update(clientId: string, options: ApiAppRequestOptions): Promise<ApiAppResponse>;
         delete(clientId: string): Promise<BaseResponse>;
     }
+
+    class HelloSign {
+        constructor(options: HelloSignOptions);
+        account: AccountModule;
+        signatureRequest: SignatureRequestModule;
+        embedded: EmbeddedModule;
+        oauth: OAuthModule;
+        team: TeamModule;
+        template: TemplateModule;
+        reports: ReportsModule;
+        unclaimedDraft: UnclaimedDraftModule;
+        apiApp: ApiAppModule;
+        setHost: (host: string, port: string, protocol: string) => void;
+        setProtocol: (protocol: string) => void;
+        setPort: (port: string) => void;
+        setClientId: (clientId: string) => void;
+        setClientSecret: (clientSecret: string) => void;
+        setOauthToken: (oauthToken: string) => void;
+        setApiKey: (key: string) => void;
+        setUserPassAuth: (username: string, password: string) => void;
+        setTimeout: (timeout: number) => void;
+        _setApiField: (key: string, value: any) => void;
+        getApiField: (key: string) => any;
+        isDev: () => boolean;
+    }
 }
 
-export = HelloSign;
+export default function hellosign(options: HelloSign.HelloSignOptions): HelloSign.HelloSign;
```

Comment:
The change to `export default` is incorrect; the implementation on npm is a CommonJS module that uses `module.exports =`. Only `export =` can be accepted here. https://unpkg.com/browse/hellosign-sdk@2.1.0/lib/hellosign.js

Reply:
I had a difficult time doing this because previously the module required HelloSign to be exported as well as the default export. 

My knowledge of CommonJS modules is limited but it seems `export =`only allows one thing from being exported from a module. Does this mean that the HelloSign cannot be exported or referenced anymore?

Reply:
Also I think you're referencing an out of date SDK. Im trying to bring this up to date with v3 of the hellosign-sdk lib https://github.com/hellosign/hellosign-nodejs-sdk/blob/v3/lib/hellosign.js